src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-list/multi-cluster-list.component.ts

```diff
@@ -189,6 +191,7 @@ export class MultiClusterListComponent {
       itemNames: [cluster['cluster_alias'] + ' - ' + cluster['user']],
       submitAction: () =>
         this.multiClusterService.deleteCluster(cluster['name'], cluster['user']).subscribe(() => {
+          this.cookieService.deleteToken(cluster['cluster_alias']);
```

Comment:
Is it possible to change the cluster name? How will it behave if that is possible?

can a cluster be logged in from two different user with same name?

Reply:
good question. I changed it and started using `fsid-username` format to store the token in cookie.